README.md

```diff
@@ -181,6 +181,7 @@ A collection of awesome browser-side [JavaScript](https://developer.mozilla.org/
 * [puppeteer](https://github.com/GoogleChrome/puppeteer) - Headless Chrome Node.js API by official Google Chrome team.
 * [webdriverio](https://github.com/webdriverio/webdriverio) - Next-gen WebDriver test automation framework for Node.js.
 * [taiko](https://github.com/getgauge/taiko) - A Node.js library with a simple API to automate Chromium based browsers.
+* [playwright](https://github.com/microsoft/playwright) - Node.js library to automate Chromium, Firefox and WebKit with a single API.
```

Comment:
If you make it `Playwright` instead of `playwright`, I see no reason to hold this PR any longer.